src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/XdocUtil.java

```diff
@@ -77,7 +80,8 @@ public static Set<Path> getXdocsConfigFilePaths(Set<Path> files) {
         final Set<Path> xdocs = new HashSet<>();
         for (Path entry : files) {
             final String fileName = entry.getFileName().toString();
-            if (fileName.startsWith("config_")) {
+            if (fileName.startsWith("config_")
```

Comment:
Will there be some point in the future where we can remove this condition?

Reply:
Yes, all `config_` files will be split so they can be removed after it is done. I added it to project board to not forget.

Reply:
We can create an actual issue on this, as we are sure that it will happen. Please do this and add 'until ...' comment so that there is no chance to forget.

Reply:
Done. Opened at https://github.com/checkstyle/checkstyle/issues/13132